src/extension/src/controller.ts

```diff
@@ -190,21 +194,30 @@ export class Controller {
     });
   }
 
+  private static getDefaultOutputPath() {
+    const userOS = os.platform();
```

Comment:
Would `os.homedir();` work better here?

Reply:
yup, for sure, I didn't know that method existed